lms/templates/student_account/account_settings.html

```diff
@@ -27,12 +30,17 @@
 
 <%block name="js_extra">
 <%static:require_module module_name="js/student_account/views/account_settings_factory" class_name="AccountSettingsFactory">
-    var fieldsData = ${ json.dumps(fields) };
-    var authData = ${ json.dumps(auth) };
-    var platformName = ${json.dumps(static.get_platform_name())};
+    var fieldsData = ${ fields | n, dump_js_escaped_json };
+    var authData = ${ auth | n, dump_js_escaped_json };
+    var platformName = ${ static.get_platform_name() | n, dump_js_escaped_json };
```

Comment:
Agreed.  You can look at the page source to see.  I had two comments, but added them in the form of adjustments to the docs.  Let me know if you figure out what to do, this being one of the things.

https://github.com/edx/edx-documentation/pull/913/commits/2d40900b8a311f7d3e42a462a821ced6a710b2b1